data/movie/I Sell the Dead (2008)/i-sell-the-dead/release.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Slug": "i-sell-the-dead",
+  "Asin": "",
+  "Upc": "",

```

Comment:
Do you have the ASIN or UPC to fill in? Doing so helps keep the quality of the data in the db high